src/d3/geo-map.d3.ts

```diff
@@ -0,0 +1,52 @@
+import * as d3 from 'd3';
+import { BaseD3, RenderOptions as BaseRenderOptions } from './base.d3';
+import * as topojson from 'topojson';
+
+export interface RenderOptions extends BaseRenderOptions {
+  topoJsonUrl: string;
+}
+
+export class GeoMapD3 extends BaseD3<RenderOptions> {
+  render() {
+    super.render();
+
+    this.renderMap().catch(console.error);
+  }
+
+  async renderMap() {
+    const { height, width, topoJsonUrl } = this.renderOptions;
+
+    const projection = d3.geoNaturalEarth1()
+      .scale(153)
```

Comment:
Magic number? Where does this come from?

Reply:
It was originally from [here](https://bl.ocks.org/da1fujimoto/abeff3fd50ba3ce4eee5dfadeb889f6b), but I ended up rewriting nearly everything including this line in #70.